### ISSUE TITLE
Show a toast/snackbar after blocking a Scanlator Group from chapter list

### ISSUE THREAD

**Issue by curche** [CarlosEsco/Neko#1165]
### Describe your suggested feature

TIL Group Blocking exists in Neko: you have to long press on a chapter and select Block Scanlator
however, this action is global: grps blocked in a single manga get blocked in other manga too
This might cause confusions in the future

the solution already exists: Settings > MangaDex > Unblock Scanlator
but this isn't very obvious

So, suggestion: have a toast or a snackbar (preferably with an Undo button) that lets user know that the grp has been blocked globally and if required, can be removed from Neko MangaDex Settings

### Other details

~inb4 people ask for Per Manga Scanlator Block~

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the newest version **[Latest](https://github.com/CarlosEsco/Neko/releases/latest)**.
- [X] I have checked through the app settings for my feature.
- [X] I will fill out all of the requested information in this form.

**Comment by CarlosEsco** [comment]
2.10.14
